9/start

```diff
@@ -6,7 +6,10 @@ WEB_CERT_FILE=${WEB_CERT_FILE:-""}
 WEB_KEY_FILE=${WEB_KEY_FILE:-""}
 
 if [ ! -z "$WEB_CERT_FILE" -a ! -z "$WEB_KEY_FILE" ]; then 
-        echo 'DAEMON_ARGS="--mlisten 127.0.0.1 -m 9390 -k '$WEB_KEY_FILE' -c '$WEB_CERT_FILE'"' >> /etc/default/openvas-gsa
+        rm -f /var/lib/openvas/CA/servercert.pem
```

Comment:
Do we really need to delete these? What if a user wants to switch back to not providing a cert file?

Reply:
Should we just move the above sed to here instead?

Reply:
Every modification to the image done in Dockerfile is persistent (like the 2 sed modifying /etc/default/openvas-gsa) while the modifications done by the start script are reverted every time you stop and start the container.

So deleting the certificate is not persistent after restarting the container. 

Moving both sed(s) modifying /etc/default/openvas-gsa (I would keep them together to avoid confusion) would have the same effect: every time you restart the container you modify openvas-gsa. It seemed to me more natural to keep these "constant" configuration in the Dockerfile, but this is more a matter of preference... functionally you could also do everything with a modified sed line in start.

Reply:
> So deleting the certificate is not persistent after restarting the container.

That's not completely true, they're persistent across starts, stops and restarts of a created container. They are not persistent across creating a fresh container but point taken and very much agreed on the matter of consistency.